### ISSUE TITLE
Additional improvements to task metrics on the admin dashboard

### ISSUE THREAD

**Issue by dhcole** [openopps/openopps-platform#895]
Following up on #882,  here are some additional improvements to the task metrics on the admin dashboard:
- [x] quarterly metrics as a separate story:  https://github.com/18F/midas/issues/905
- [ ] unique participants
- [ ] consolidate active opportunities in the top section with the new tables (so we can see task creators and total active users (unique(task creators + participants)
- [ ] break out opportunities by type https://github.com/18F/midas/issues/1006 (depends on #872)
- [ ] task creators over time
- [ ] agencies with participants over time
- [ ] new agencies over time
- [ ] repeat task takers
- [ ] implement goals

**Comment by ultrasaurus** [comment]
FYI: added quarterly metrics as a separate story:  https://github.com/18F/midas/issues/905

**Comment by dhcole** [comment]
Also....

- [ ] task creators over time
- [ ] agencies with participants over time

**Comment by dhcole** [comment]
- [ ] new agencies over time

**Comment by ultrasaurus** [comment]
[ ] repeat task takers -- this deserves a little mockup, but adding it so we remember

**Comment by LisaLNelson** [comment]
Complete.  Made separate note of metric desired.
